backend/stories/schemas.py

```diff
@@ -33,15 +38,22 @@ class CreateStory(BaseModel):
     current_location: str
     sickness_start: str
 
+
 class Story(CreateStory):
     id: int
+    token: str
+    user: User = None
```

Comment:
Actually I added this for testing purposes (to check that users and stories were being associated) but it might aswell go away